core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -71,10 +72,11 @@
     private Factory() {} // prevent instantiation
 
     public static FileSystem get(AlluxioConfiguration alluxioConf) {
-      return get(FileSystemContext.create(alluxioConf));
+      return get(ClientContext.create(alluxioConf));
     }
 
-    public static FileSystem get(FileSystemContext context) {
+    public static FileSystem get(ClientContext context) {
+      Preconditions.checkNotNull(context);
```

Comment:
when using `Preconditions.checkNotNull`, include the name of the variable so that if the check fails, it can help us find where the issue occurred.
```java
Preconditions.checkNotNull(context, "context");
```